asteroid/data/conferencing_speech_datastet.py

```diff
@@ -0,0 +1,319 @@
+import numpy as np
+import math
+import soundfile as sf
+import scipy.signal as sps
+import librosa
+import os
+import warnings
+from torch.utils.data import Dataset
+import torch
+
+eps = np.finfo(np.float32).eps
+
+
+class CS2021(Dataset):
+    """Dataset class for LibriMix source separation tasks.
+
+    Args:
+        config_dir (str): The path to the config file.s
+        rir_type (str): One of ``'linear'``, ``'circle'``, ``'non_uniform'``
+        sample_rate (int) : The sample rate of the sources and mixtures.
+        segment (int, optional) : The desired sources and mixtures length in s.
+
+    """
+
+    dataset_name = "CS2021"
+
+    def __init__(
+        self,
+        config_dir,
+        rir_type="linear",
```

Comment:
because FasNetTAC is array topology independent here we can use a list of RIRs also to train on everything. 
I will add this